Sources/SwifterSwift/SwiftStdlib/SequenceExtensions.swift

```diff
@@ -341,24 +341,24 @@ public extension Sequence where Element: Equatable {
     ///        ["h", "e", "l", "l", "o"].contains(["l", "o"]) -> true
     ///
     /// - Parameter elements: array of elements to check.
-    /// - Returns: true if array contains all given items.
+    /// - Returns: true if sequence contains all given items.
     /// - Complexity: _O(m·n)_, where _m_ is the length of `elements` and _n_ is the length of this sequence.
-    func contains(_ elements: [Element]) -> Bool {
+    func contains<S: Sequence>(_ elements: S) -> Bool where S.Element == Self.Element {
```

Comment:
Remove `Self` and match the other function (plus it's how Apple does it)
```suggestion
    func contains<S: Sequence>(_ elements: S) -> Bool where Element == S.Element {
```

Reply:
Actually Apple would do it
```suggestion
    func contains<S>(_ elements: S) -> Bool where S: Sequence, Element == S.Element
```
but you can do it however you like.